boost/meta.yaml

```diff
@@ -17,6 +17,8 @@ requirements:
     - libpython         [win]
```

Comment:
What do you mean? The first line in build dependencies is `python`. Nevertheless, I haven't touched Windows-specific configs at all. I tested everything on CentOS 6.6, Ubuntu 14.04, OS X, though.